ipc/ipc.go

```diff
@@ -156,10 +156,10 @@ func (env *Env) Close() error {
 // failed: true if executor has detected a kernel bug
 // hanged: program hanged and was killed
 // err0: failed to start process, or executor has detected a logical error
-func (env *Env) Exec(p *prog.Prog) (output []byte, cov [][]uint32, errnos []int, failed, hanged bool, err0 error) {
+func (env *Env) Exec(p *prog.Prog, pid int) (output []byte, cov [][]uint32, errnos []int, failed, hanged bool, err0 error) {
```

Comment:
pid should be an argument to ipc.MakeEnv.
We've discussed that we may need to pass pid to executor to setup network card, etc.
MakeEnv looks like the right place for pid long term.